Fundamentals/Backoffice/Property-Editors/Built-in-Property-Editors/Media-Picker/index.md

```diff
@@ -1,20 +1,24 @@
 ---
-versionFrom: 9.0.0
-versionTo: 10.0.0
+versionFrom: 8.1.0
+versionTo: 8.14.0
```

Comment:
Should this be 8.13.0? Since the note below mentions 8.14 and links to the new media picker? It can be a little confusing if the versionTo has 8.14.0 cz it means the article is valid for it but then the note says otherwise.

Reply:
Yep. That's a great point! I'll update it to 8.13.0 💪